RNTrueLayerPaymentsSDK/js/models/theme/Theme.ts

```diff
@@ -11,6 +11,6 @@ export type Theme = Partial<{
   ios: Partial<{
     lightColors: iOSColors;
     darkColors: iOSColors;
-    fontFamilyName: string;
+    fontFamilyName?: string;
```

Comment:
Would like mobile guys to take a look at this. This was required to get it to build, but absolutely could be wrong.